scripts/mem_usage.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python
+#
+# Copyright (c) 2014-2017, Linaro Limited
+# All rights reserved.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+import argparse
+import os
+import re
+import subprocess
+import sys
+
+def get_args():
+
+	parser = argparse.ArgumentParser(
+		description='Shows the memory layout of the TEE ELF file. '
+		'Size is provided for each section.')
+	parser.add_argument('tee_elf', help='The OP-TEE ELF file (tee.elf)')
+	return parser.parse_args()
+
+def printf(format, *args):
+	sys.stdout.write(format % args)
+
+def print_sect(name, addr, size, round_up, print_num_pages):
```

Comment:
You can define it as `print_sect(name, addr, size, round_up = False, print_num_pages = False)` to reduce number of parameters in default case.

Reply:
OK